docs/release/release-tasks.md

```diff
@@ -0,0 +1,384 @@
+  the corresponding role are not responsible to do it themselves. 
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Release Lead](#release-lead)
+  - [Responsibilities](#responsibilities)
+  - [Tasks](#tasks)
+    - [Set a tentative release date for the minor release](#set-a-tentative-release-date-for-the-minor-release)
+    - [Assemble release team](#assemble-release-team)
+    - [Finalize release schedule and team](#finalize-release-schedule-and-team)
+    - [Prepare main branch for development of the new release](#prepare-main-branch-for-development-of-the-new-release)
+    - [Create a new GitHub milestone](#create-a-new-github-milestone)
+    - [[Track] Remove previously deprecated code](#track-remove-previously-deprecated-code)
+    - [[Track] Bump dependencies](#track-bump-dependencies)
+    - [Create a release branch](#create-a-release-branch)
+    - [[Continuously] Maintain the GitHub release milestone](#continuously-maintain-the-github-release-milestone)
+    - [[Repeatedly] Cut a release](#repeatedly-cut-a-release)
+    - [[Optional] [Track] Bump the ClusterAPI apiVersion](#optional-track-bump-the-clusterapi-apiversion)
+    - [[Optional] [Track] Bump the Kubernetes version](#optional-track-bump-the-kubernetes-version)
+- [Communications/Docs/Release Notes Manager](#communicationsdocsrelease-notes-manager)
+  - [Responsibilities](#responsibilities-1)
+  - [Tasks](#tasks-1)
+    - [Add docs to collect release notes for users and migration notes for provider implementers](#add-docs-to-collect-release-notes-for-users-and-migration-notes-for-provider-implementers)
+    - [Update supported versions](#update-supported-versions)
+    - [Ensure the book for the new release is available](#ensure-the-book-for-the-new-release-is-available)
+    - [Polish release notes](#polish-release-notes)
+    - [Change production branch in Netlify to the new release branch](#change-production-branch-in-netlify-to-the-new-release-branch)
+    - [Update clusterctl links in the quickstart](#update-clusterctl-links-in-the-quickstart)
+    - [Continuously: Communicate key dates to the community](#continuously-communicate-key-dates-to-the-community)
+- [CI Signal/Bug Triage/Automation Manager](#ci-signalbug-triageautomation-manager)
+  - [Responsibilities](#responsibilities-2)
+  - [Tasks](#tasks-2)
+    - [Setup jobs and dashboards for a new release branch](#setup-jobs-and-dashboards-for-a-new-release-branch)
+    - [[Continuously] Monitor CI signal](#continuously-monitor-ci-signal)
+    - [[Continuously] Reduce the amount of flaky tests](#continuously-reduce-the-amount-of-flaky-tests)
+    - [[Continuously] Bug triage](#continuously-bug-triage)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Release Lead
```

Comment:
isn't some of this duplicate of https://github.com/kubernetes-sigs/cluster-api/pull/7499/files#diff-880d57dd8c9aaa6dc1498d94d07a9a1c0b34b56f254b5ca011785cf702599d29L49 ? I like this version better, but maybe we should keep all of this in the release-team doc and link it to from here instead?

Reply:
The link doesn't work for me but I assume it points to release-team.md.

>  but maybe we should keep all of this in the release-team doc and link it to from here instead?

I had similar thoughts. I can merge the responsibilities back into the release-team doc, currently the release-team doc doesn't have the separation in roles.

What I liked about having it here in release-tasks.md is that it provides context for all the tasks that belong to the role (and also makes the gaps more visible between the responsiblities and what is covered directly via tasks).


I would probably prefer keeping responsibilities here and making the ones in release-team a bit more high-level so we don't duplicate the info and don't have to keep them in sync. (I would then link from release-team.md to release-tasks.md for the full details)

Reply:
Added a note to release-team.md